defi/src/protocols/data3.ts

```diff
@@ -42719,12 +42719,12 @@ const data3_2: Protocol[] = [
     description: "Origami is a first-in-class automated leverage protocol launching on Ethereum and 🐻",
     chain: "Ethereum",
     logo: `${baseIconsUrl}/origami-finance.jpg`,
-    audits: "2",
+    audits: "17",
```

Comment:
please revert this change, this value isnt the number of audits

Reply:
Oh - what does it refer to?

Reply:
reverted

Reply:
/* Audits: Please follow this legend
0 -> No audits
1 -> Part of this protocol may be unaudited
2 -> Yes
3 -> This protocol is a fork of an existing audited protocol